src/modules/Security.rb

```diff
@@ -772,8 +805,9 @@ def Import(settings)
           end
         end
       end
-      @Settings = tmpSettings
 
+      @Settings = tmpSettings
+      set_selinux_patterns # Checking needed packages
```

Comment:
this looks strange. I think autoyast has own method for required packages and not use this PackagesProposal...which I think is not used if confirm mode is set to false.

Reply:
I didn't know :(

Anyway, since this is "just" a sync PR, lets merge this as it is and fix it first in SP2 and then move it forward to SP3, Do you agree?

Reply:
fine for me.